lib/openscap_parser/value_definition.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class ValueDefinition < XmlNode
+    include OpenscapParser::Util
+
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def description
+      @description ||= newline_to_whitespace(
+        parsed_xml.at_css('description') &&
+          parsed_xml.at_css('description').text || ''
+      )
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title') &&
+        parsed_xml.at_css('title').text
+    end
+
+    def type
+      @type ||= parsed_xml['type'] || 'string'
+    end
+
+    def lower_bound
+      lower_bound_element = parsed_xml.at_xpath("lower-bound[@selector='']") || parsed_xml.at_xpath('lower-bound[not(@selector)]')
```

Comment:
It's possible for there to be more than one `lower-bound` and `upper-bound` elements in a Value definition so we need to make sure we're returning the default `lower-bound`/`upper-bound` numbers here. 
The default element is the one without an empty or absent @selector, so that's what we're getting here.

But this just made me realize that when a profile uses a refined value that has the value type "number", we'll need to check if there's a non-default `lower-bound`/`upper-bound' with the same `@selector` attribute.. So we'll also need to parse non-default `lower-bound`/`upper-bound' in another method here too I think

Reply:
How woult this look like if we don't deal with refinements?

Reply:
This would still look like this because we still need to make sure we're getting the default lower/upper bounds which is element that has an empty `selector` attribute or no `selector` attribute